run-build.ps1

```diff
@@ -68,8 +68,8 @@ $env:VSTEST_TRACE_BUILD=1
 # install a stage0
 $dotnetInstallPath = Join-Path $RepoRoot "scripts\obtain\dotnet-install.ps1"
 
-Write-Output "$dotnetInstallPath -Channel ""master"" -InstallDir $env:DOTNET_INSTALL_DIR -Architecture ""$Architecture"""
-Invoke-Expression "$dotnetInstallPath -Channel ""master"" -InstallDir $env:DOTNET_INSTALL_DIR -Architecture ""$Architecture"""
+Write-Output "$dotnetInstallPath -Channel ""master"" -version ""2.1.0-preview1-007172"" -InstallDir $env:DOTNET_INSTALL_DIR -Architecture ""$Architecture"""
```

Comment:
Why can't this just be the latest from master?

Reply:
Because that's the version for which I have built the bootstrap CLI and put it to Azure. Is there a problem with that?

Reply:
Not at all. Just wanted to confirm. We can unpin this once a build is produced.

Can you please, also work with @johnbeisner to review the install script and to stand-up a VSO official leg for it?